src/main/java/org/wso2/carbon/module/swiftiso20022/MT940FormatValidator.java

```diff
@@ -138,6 +138,11 @@ private Map<String, Object> extractFields(String[] lines) {
             if (line.startsWith(OPEN_BRACKET) || line.startsWith(DASH) || line.length() < 3) {
                 continue;
             }
+
+            if (line.endsWith("&#xd;")) {
```

Comment:
Shall we add a comment here mentioning why we are doing this

Reply:
Added in e2eba5896d2549c41ebd26848c9e0af9148a62ce